crates/electrum/src/electrum_ext.rs

```diff
@@ -223,9 +204,8 @@ impl ElectrumExt<ConfirmationHeightAnchor> for Client {
 
         let update = loop {
             let (tip, _) = construct_update_tip(self, prev_tip.clone())?;
-            let mut update = ElectrumUpdate::<K, ConfirmationHeightAnchor>::new(tip.clone());
-            let cps = update
-                .new_tip
+            let mut graph_update = IncompleteTxGraph::<ConfirmationHeightAnchor>::default();
```

Comment:
In general can we avoid the term "graph" in this code (unless we are actually returning a `TxGraph`). We are just finding relevant transactions to set of script pubkeys.